vertical_integration/FORCE_Components/ComponentSetsFiles/Sets1/ReadMe

```diff
@@ -0,0 +1,36 @@
+# In this folder, each file that starts with "Setfile" is a file that the user can create or modify to decide which components needs to be collected together as a component set.
+
+# The file set can have any of the following three forms:
+
+1- Grouping components that have the same category (e.g. all the turbines, all the pumps, ...etc).
+For example, the example below means that all the components that are considered pumps will be grouped together. Note that the category of each component is extracted from the Aspen HYSYS output XLSX file.
+
+Example:
+
+{
+  "Set Name": "pumps set",
+  "Included Categories": ["Pumps"]
+}
+
+
+2- Grouping components by adding the components names manually. The names of the components are extracted from the Aspen HYSYS output XLSX file.
+
+Example:
+
+{
```

Comment:
It would be best if you made the examples render as code  blocks you can do that by surrounding the chunk in three backticks (```)

Reply:
Done